uiplib/settings.py

```diff
@@ -2,8 +2,8 @@
 import json, argparse
 
 HOME_DIR = os.path.join(os.path.expanduser("~"), '.uip')
-NUMBER_OF_IMAGES_TO_PARSE = 24
-DEFAULT_PICS_FOLDER = os.path.join(HOME_DIR, 'pics')
+NUMBER_OF_IMAGES_TO_PARSE = 15
+DEFAULT_PICS_FOLDER = os.path.join(os.path.expanduser("~"), 'pics')
```

Comment:
Don't change HOME_DIR, bro. We need this to hide the downloaded images. As @abhsag24 already mentioned in the general channel